index.rst

```diff
@@ -802,77 +682,105 @@ Again, you will need to replace ``<Component>`` by the name of the CSC.
-        
-	ssh -L 5900:192.168.1.49:5900 saluser@139.229.162.118
-
-#. Using RealVNC (which is required due to encryption although other clients might work) you can then connect to 'localhost' on port 5900
+#. Using RealVNC (which is required due to encryption although other clients
+    might work) connect to 'atmcs-dev.cp.lsst.org' on port 5900
 #. Enter credentials (ask Patrick or Tiago)
-#. If the GUI is not already open, then open internet explorer and enter the following address in the address bar.
+#. If the GUI is not already open, then open internet explorer and enter the
+    following address in the address bar.
    
    .. code-block:: python 
         
-	http://192.168.1.47:8000/atmcs.html
+	http://atmcs-crio.cp.lsst.org:8000/atmcs.html
 
-One can also install the `LabVIEW remote panel <http://ftp.ni.com/support/softlib/labview/labview_runtime/2010/2010Sp1%20Linux%20Temp/labview-2010-rte-10.0.1-1.i386.zip>`_ on their Windows machine (Internet Explorer only) then open a tunnel to the above IP on port 8000. This requires the download from NI, then you'll have to open the tunnel using PuTTy (or equivalent). Details will be included in the ATMCS documention upon delivery. We don't recommend this method unless absolutely necessary.
+One can also install the `LabVIEW remote panel <http://ftp.ni.com/support/softlib/labview/labview_runtime/2010/2010Sp1%20Linux%20Temp/labview-2010-rte-10.0.1-1.i386.zip>`_
+on their Windows machine (Internet Explorer only) then open a tunnel to the
+above IP on port 8000. This requires the download from NI. Details will be
+included in the ATMCS documentation upon delivery. We don't recommend this
+method unless absolutely necessary.
 
 .. _hexapod_connection_reset:
 
 Resetting the ATHexapod IP Connection
 -------------------------------------
 
-For reasons which are under investigation, occasionally after a power cycle (we think) the hexapod TCP/IP connection goes down. To reset it, one must connect a serial port to the device, establish a connection using the (windows) PIMikroMove software, close the connection, then power cycle the controller. Power cycling can be done remotely (:ref:`using the switched PDU <telescope_cabinet_pdu>`). Until this problem is resolved, we've left a permanent serial (RS-232) connection to a local windows machine.
+For reasons which are under investigation, occasionally after a power cycle
+(we think) the hexapod TCP/IP connection goes down. To reset it, one must
+connect a serial port to the device, establish a connection using the (windows)
+PIMikroMove software, close the connection, then power cycle the controller.
+Power cycling can be done remotely (:ref:`using the switched PDU
+<telescope_cabinet_pdu>`). Until this problem is resolved, we've left a
+permanent serial (RS-232) connection to a local windows machine.
 
 Follow these steps to re-establish TCP/IP connection:
 
 #. Establish VNC connection :ref:`which is the same as the ATMCS GUI VNC shown here <atmcs_gui>`.
```

Comment:
This is now wrong, you have to open a teamviewer to Mike Warner's PC.

Reply:
Can you update this in a different ticket? not sure I know what to do now... I'll make a note that this needs updating.

Reply:
Sure